src/elm/Main.elm

```diff
@@ -2597,6 +2609,20 @@ viewBuildsFilter shouldRender org repo maybeEvent =
         text ""
 
 
+viewTimeToggle : Bool -> Bool -> Html Msg
+viewTimeToggle shouldRender showTimestamp =
+    if shouldRender then
+        div [ class "form-controls", class "-stack", Html.Attributes.style "justify-content" "space-around" ]
```

Comment:
we should use some form of css class instead of inline style
https://webhint.io/docs/user-guide/hints/hint-no-inline-styles/

```suggestion
        div [ class "form-controls", class "-stack", class "time-toggle" ]
```